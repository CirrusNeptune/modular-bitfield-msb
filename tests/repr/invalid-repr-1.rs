use modular_bitfield_msb::prelude::*;

#[bitfield]
#[repr(invalid)]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
