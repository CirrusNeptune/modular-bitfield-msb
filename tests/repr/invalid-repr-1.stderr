error[E0552]: unrecognized representation hint
 --> $DIR/invalid-repr-1.rs:4:8
  |
4 | #[repr(invalid)]
  |        ^^^^^^^

error[E0692]: transparent struct cannot have other repr hints
 --> $DIR/invalid-repr-1.rs:4:1
  |
4 | #[repr(invalid)]
  | ^^^^^^^^^^^^^^^^
